django_components/types.py

```diff
@@ -0,0 +1,9 @@
+try:
+    from typing import Annotated
+except ImportError:
+    from typing_extensions import Annotated
```

Comment:
This needs to be added as a dependency to avoid crashes I think?

Reply:
I removed typing-extensions and instead created a simple Annotated class for the cases of Python <3.9. I checked and it still works well to provide the syntax highlighting.

I think it's better not to add additional dependencies if not really necessary, especially for this small use cases.